mobile-app/integration_test/news_test.dart

```diff
@@ -61,33 +61,33 @@ void main() {
       await tester.pumpAndSettle();
       await binding.takeScreenshot('news-bookmark-feed');
 
-      // Check if article is in bookmark view and it has same title and author
-      final Finder bookmarkArticle =
+      // Check if tutorial is in bookmark view and it has same title and author
+      final Finder bookmarkTutorial =
           find.byKey(const Key('bookmark_article_0'));
-      final Finder bookmarkArticleText = find.descendant(
-        of: bookmarkArticle,
+      final Finder bookmarkTutorialText = find.descendant(
+        of: bookmarkTutorial,
         matching: find.byType(Text),
       );
       expect(
-        tester.firstWidget<Text>(bookmarkArticleText.first).data!,
+        tester.firstWidget<Text>(bookmarkTutorialText.first).data!,
         title.data,
       );
       expect(
-        (tester.firstWidget<Text>(bookmarkArticleText.last).data!)
+        (tester.firstWidget<Text>(bookmarkTutorialText.last).data!)
             .split('Written by: ')[1],
         author.split('Written by ')[1],
       );
 
       // Check database if record exists
       final db = await openDatabase(
-          path.join(await getDatabasesPath(), 'bookmarked-article.db'));
+          path.join(await getDatabasesPath(), 'bookmarked-tutorial.db'));
```

Comment:
I think we want this to stay the same. This will break the already saved bookmarks on the next update